packages/typechain/test/cli/parseArgs.test.ts

```diff
@@ -62,4 +62,36 @@ describe('cli > parseArgs', () => {
       flags: { alwaysGenerateOverloads: true },
     })
   })
+
+  it('shows usage guide given --help flag', () => {
+    process.argv = ['', '', '--help']
+
+    const processExit = process.exit
+    // eslint-disable-next-line no-console
+    const consoleLog = console.log
+
+    const exitMock = mockFn<typeof process.exit>(() => {
+      throw new Error('process.exit called')
+    })
+    process.exit = exitMock
+    const logMock = mockFn<typeof console.log>(() => {})
+    // eslint-disable-next-line no-console
+    console.log = logMock
+
+    try {
+      parseArgs()
+    } catch (err) {
+      expect((err as Error).message).toEqual('process.exit called')
```

Comment:
why not use: `expect().toThrow('process.exit called')`?
 
 https://earljs.dev/docs/guides/testing-errors#matching-only-part-of-an-error-message